### ISSUE TITLE
Tab Component: a11y : aria-expanded attribute is missing to indicate where a control is collapsed or expanded. 

### ISSUE THREAD

**Issue by yoganjan** [SAP/ui5-webcomponents#9958]
### Describe the bug

For the tab component , when sub tabs are available, the aria-expanded attribute is missing to indicate where a control is collapsed or expanded. The attribute value should change to true when a control is expanded and be set to false when it is collapsed. The attribute is required, whether it is true or false,

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/layouts-floorplans-tabcontainer--with-nested-tabs

### Reproduction steps

1. Navigate to https://sap.github.io/ui5-webcomponents-react/v2/?path=/story/layouts-floorplans-tabcontainer--with-nested-tabs
2. Observe that the Tab component has sub tabs
3. When the sub tabs are opened or closed , aria-expanded attribute is missing

### Expected Behaviour

The aria-expanded attribute value should change to true when a control is expanded and be set to false when it is collapsed.

### UI5 Web Components for React Version

v2

### UI5 Web Components Version

v2

### Browser

Chrome

### Operating System

Mac

### Organization

SAp

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by LidiyaGeorgieva** [comment]
Hello @yoganjan ,

The tabs are not collapsible/expandable. They just open a menu with subtabs, which is announced.
I see no issue here.

Best Regards,
Lidiya
